ApolloExtra.lua

```diff
@@ -2551,32 +2759,60 @@ OlympusLootData.ApolloUpgrade = {
 				RequiredTextLines = { "ApolloFirstPickUp", "ApolloAboutPersephoneMeeting01" },
 				RequiredFalseTextLines = { "OlympianReunionQuestComplete" },
 				RequiresLastRunCleared = true,
-				{ Cue = "/VO/Apollo_0180",
+				{ Cue = "/VO/Apollo_0079",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Oh what is going {#DialogueItalicFormat}on {#PreviousFormat}down there, dearest? I thought for certain you would make it out, but seems that nasty place has kept you grounded, hasn't it. But don't despair, and you take heart, won't you?" },
+					Text = "What are you doing there, Zagzag. I saw in a vision that you escaped... So why did you went back? Forgot something, maybe? Oh well, I also saw that you would do it again, so no worries." },
```

Comment:
"What are you doing there, Zagzag? I had a vision that you escaped... So why did you go back? Forgot something, maybe? Oh well, I also saw that you would escape again, so no worries."